### ISSUE TITLE
Change button to layout like M8.run

### ISSUE THREAD

**Issue by illtscha** [v3rm0n/m8c-android#162]
Hi!
First off sorry I have a little programming background with python and html/CSS/java but I have never tried making android apps so I have no idea if that's complicated or not.

My question is if it possible to make an option to use a layout like the real deal? I mean something like they got on m8.run if you enable show controls. Its really hard to work with the controls right now since they are really small and very close together.

Thank you!

**Comment by v3rm0n** [comment]
Yeah it was very quickly put together. Can you share a screenshot of how it looks for you? Devices are very different so it would be good to understand the problem a bit better.

**Comment by v3rm0n** [comment]
@illtscha try out this version, there's a new option to toggle in the UI settings https://github.com/v3rm0n/m8c-android/releases/tag/v1.0.61

**Comment by illtscha** [comment]
Its sooo much better now than before!! Thank you very much!
What would be great thought would be an option to scale them because theyre still a bit too small for my taste.
Thank you for your great work!!!!

**Comment by mrtnvgr** [comment]
Also, it would be cool if button color would change based on the current theme. Is it possible to implement?
